### ISSUE TITLE
Support legacy junit xml from pytest 

### ISSUE THREAD

**Issue by john-dupuy** [ibutsu/ibutsu-server#92]
Recently importing this Junit XML file https://github.com/SatelliteQE/betelgeuse/blob/master/sample_project/results/sample-junit-result.xml failed with the traceback:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/app/ibutsu_server/tasks/__init__.py", line 30, in __call__
    return super().__call__(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 648, in __protected_call__
    return self.run(*args, **kwargs)
  File "/app/ibutsu_server/tasks/importers.py", line 76, in run_junit_import
    for testsuite in tree.testsuite:
  File "src/lxml/objectify.pyx", line 231, in lxml.objectify.ObjectifiedElement.__getattr__
  File "src/lxml/objectify.pyx", line 450, in lxml.objectify._lookupChildOrRaise
AttributeError: no such child: testsuite
```
This appears to be due to a legacy pytest xml version https://docs.pytest.org/en/stable/reference.html?highlight=junit#confval-junit_family.

Ideally we should support both. At the very least we should specify which XML version we support.

**Comment by jjaquish** [comment]
That MR I have open should solve this issue.

**Comment by rsnyman** [comment]
Fixed in #150
